static/js/stream_data.js

```diff
@@ -345,6 +345,17 @@ exports.add_subscriber = function (stream_name, user_id) {
     return true;
 };
 
+exports.remove_deactivated_user_from_all_streams = function (user) {
+    (stream_info.values()).forEach(function (stream) {
+        if (exports.user_is_subscribed(stream.name, user.email)) {
```

Comment:
Might be better to do a small `people.js` refactor to make a `exports.user_is_subscribed_by_id` rather than going through the user's email; we are trying to migrate away from unnecessary use of email addresses in the Zulip JS codebase.